src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -26,6 +26,10 @@
  */
 public class CommandTestUtil {
 
+    public static final String VALID_CODE_CS3216 = "CS3216";
+    public static final String VALID_CREDITS_CS3216 = "5";
+    public static final String VALID_TAG_NON_CORE = "nonCore";
+
```

Comment:
I added module field templates here and left the rest of the templates alone. We can remove at the end if nobody else needs them